### PR TITLE
test: add ar-related components

### PR DESCRIPTION
**Pull request by WoodNeck**
This adds total 526 tests.
Estimated coverage is 68.8%
image

image

### CODE REVIEW

**Review by mixed — APPROVED** [review]
image

File: package.json [hunk 1]
```diff
@@ -1,86 +1,105 @@
...
+    "typescript"
+  ],
+  "author": "NAVER Corp.",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/naver/egjs-view3d/issues"
+  },
+  "homepage": "https://github.com/naver/egjs-view3d#readme",
+  "dependencies": {
+    "chai-as-promised": "^7.1.1",
```
Comment (daybrush) [inline package.json:44]: change to dev dependencies.

**Review by daybrush — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (WoodNeck) [reply package.json:44]: Thanks for checking 👍

**Pull request merged**